tardis/montecarlo/setup_package.py

```diff
@@ -30,14 +31,16 @@ def get_extensions():
     deps += [os.path.relpath(fname) for fname in glob(
         os.path.join(os.path.dirname(__file__), 'src/randomkit', '*.h'))]
 
-    return [Extension('tardis.montecarlo.montecarlo', sources,
-                      include_dirs=['tardis/montecarlo/src',
-                                    'tardis/montecarlo/src/randomkit',
-                                    np.get_include()],
-                      depends=deps,
-                      extra_compile_args=compile_args,
-                      extra_link_args=link_args,
-                      define_macros=define_macros)]
+    return cythonize(
```

Comment:
That wouldn't do anything since this code is only run when the `setup.py` is run.  In any case adding cythonize here doesn't do anything regardless, since `astropy_helpers.commands.build_ext` does this for you automatically.

Something I've wanted to do for a while but has been low priority is to enhance [this code](https://github.com/astropy/astropy/blob/v1.1/astropy/__init__.py#L195) (there's a version of this in the affiliated package template too) so that it automatically rebuilds extension modules any time one of them has been changed.

In the meantime, if you'd doing development on extension modules you have to manually run `./setup.py build_ext --inplace`.